Fixed misused sqllogging. Closes #26427

Megmeehey opened the pull request:
Hey, guys!

You have `sqllogging`, which usage was misplaced. This PR fixes that, and it turns out, that `sqllogging` is actually never used at all.

IDK, how do I properly generate global vars file, or fix MD5 in run_test.sh, but I did 
```
python3 tools/GenerateGlobalVarAccess/gen_globals.py baystation12.dme code/_helpers/global_access.dm
```
and then just copied MD5 to run_test.sh

Closes #26427

afterthought2 reviewed (dismissed):
This is fine, but I need to make sure this config var is actually set properly on the live server. Do not merge.

MistakeNot4892 commented:
That is the correct process for changing globals, for the record.

Megmeehey commented:
Also removed ENABLE_STAT_TRACKING from example config, as it was used to set `sqllogging` and now obsolete.

afterthought2 commented on code/procs/dbcore.dm:
```diff
@@ -56,17 +56,19 @@ DBConnection/New(dbi_handler,username,password_handler,cursor_handler)
 
 DBConnection/proc/IsConnected()
-	if(!sqllogging) return 0
+	if(!config.sql_enabled)
```
Please move the import of this "new" (currently unused) config var to the db config file, and include it in the example db config in place of the misnamed one you're removing.

Megmeehey replied:
Moved in 8e7022f. Now its more like a rename 😮

Megmeehey replied:
Sorry it took so long tho.

Commit:
Fixed misused sqllogging. Closes #26427
Removed ENABLE_STAT_TRACKING from example config
Moved SQL_ENABLED to db config file

Megmeehey commented:
Kicked travis by removing empty line. Thanks to byond.com outage, it could not download needed archive.

afterthought2 reviewed (approved):
Looks OK, will look into merging when I have time. Don't merge.
